code/game/machinery/computer/crew.dm

```diff
@@ -62,6 +62,8 @@ GLOBAL_DATUM_INIT(crewmonitor, /datum/crewmonitor, new)
 	jobs["Mime"] = 67
 	jobs["Janitor"] = 68
 	jobs["Lawyer"] = 69
+	jobs["Barber"] = 70
+	jobs["Stage Magician"] = 71

```

Comment:
I'm not fond of having to hardcode every gimmick job in various places throughout the codebase but refactoring this is sadly out of the scope of this PR.

Reply:
that's what i tried to do initially when you requested one preference,  but it was a bit out of my league